packages/astro/src/core/errors/errors-data.ts

```diff
@@ -620,8 +620,42 @@ export const ExpectedImageOptions = {
 	message: (options: string) =>
 		`Expected getImage() parameter to be an object. Received \`${options}\`.`,
 } satisfies ErrorData;
+
+/**
+ * @docs
+ * @see
+ * - [Images](https://docs.astro.build/en/guides/images/)
+ * @description
+ * Astro could not find an image you imported. Usually, this is simply caused by a typo in the path.
```

Comment:
```suggestion
 * Astro could not find an image you imported. Often, this is simply caused by a typo in the path.
```
I do like hedging our bets a little more with "often" here!